File: APYcalculator.c
#include <stdio.h>

int main() {
    float yield;
    float startsum;
    int period;
    int i;
    float cal;

    printf_s("Enter the starting sum:\n");
    scanf_s("%f", &startsum);
    printf_s("Enter the yield:\n");
    scanf_s("%f", &yield);
    printf_s("Enter the period:\n ");
    scanf_s("%d", &period);

    for(i = 0; i < period; i++){
       cal = startsum + (startsum * (yield / 100));
       startsum = cal;
 }

    printf_s("Total earned for the period is: ", cal);
    printf_s("%.2f\n", cal);
}

//do a calculator for APY.

File: Python-Basics/APYcalculator.py
# This is a calculator with which you can see the real rate of return earned on an investment,
# taking into account the effect of compounding interest.

start_sum = float(input('Enter the starting sum :\n'))
period = int(input('Enter the period in years :\n'))
yield1 = float(input('Enter the yield :\n'))
x = float()
i = 0

for i in range(period):
    i += 1
    x = float(start_sum + (start_sum * (yield1 / 100)))
    start_sum = x

print('Total earned :', x)
